omnisafe/adapter/onpolicy_adapter.py

```diff
@@ -109,9 +109,6 @@ def rollout(  # pylint: disable=too-many-locals
                     last_value_c = torch.zeros(1)
                     if not done:
                         if epoch_end:
-                            logger.log(
```

Comment:
why cancel this?

Reply:
Because Safe Isaac Gym initialized 1024+ environment, this `print` information is pretty messy.